src/main/java/seedu/address/logic/commands/FindMeetingCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Finds and lists all persons in address book who are marked as high priority.
```

Comment:
maybe you can modify the java document according to the class FindMeetingCommand